docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -1243,6 +1243,17 @@ List of QoS Policy data members:
      |BuiltinAttributes::discovery_config-api| within |WireProtocolConfigQos::builtin-api| (see
      :ref:`DS_modify_server_list`).
 
+.. note::
+     In single process deployments where multiple Datareaders and Datawriters are created within the same
+     DomainParticipant, each of them will have the same **participantId** but different *unicast locator* ports.
+     That can lead on participant creation failure if the amount of Datareaders and Datawriters to be created reaches
+     the value of
+     :cpp:var:`BuiltinAttributes::mutation_tries<eprosima::fastdds::rtps::BuiltinAttributes::mutation_tries>`, due to
+     exceeding the maximum amount of mutations of the same **participantId** into different port number for the
+     *unicast locator* creation.
```

Comment:
This is not true. The issue is when creating participants in different processes.
Each participant will start trying the lowest unicast port for its domain, and then try successive unicast ports.
It will also fail to find a unicast port when creating more than 100 participants in a single process, and then create a participant on a new process. The latter won't find an available unicast port unless increasing mutation_tries